### DISCUSSION TITLE
Field type for list of doc-references

### DISCUSSION THREAD

**Discussion by webwurm** [AntlerVC/firetable#845] [Support / Q&A] [answered]
I have a question about a field type and if this is possible in Rowy.

I have "questions" with "tags". The questions are in a collection; but since a tag has more information (category etc.) I have it in a seperate collection and just refer to it, like so:
![Tags](https://user-images.githubusercontent.com/1053176/192517943-6933bb2a-887c-4f19-b8b8-6d5a5854a8b4.jpg)

How do I set the "tags"-field of the "question"-collection up in Rowy?

Thanks a lot!

**Comment by notsidney** [comment]
Hi @webwurm, you can use the JSON field for this use case. See instructions here: https://github.com/rowyio/rowy/discussions/731

**Comment by webwurm** [comment, marked as answer]
Thanks for the reply, I tried that, but the field stays empty:
<img width="278" alt="Tags 01" src="https://user-images.githubusercontent.com/1053176/195013500-b7d08a4f-40a3-4308-8f25-561e9340126d.png">
<img width="204" alt="Tags 02" src="https://user-images.githubusercontent.com/1053176/195013503-af0c2ebd-c548-4ad4-847c-40f5c980a2d6.png">

**Reply by webwurm** [reply]
Also, when I select the row it sometimes happens that Rowy crashes:
<img width="535" alt="Tags 03" src="https://user-images.githubusercontent.com/1053176/195014014-907816ad-c7b6-4cd7-aa13-221c4e7289eb.png">

**Reply by notsidney** [reply]
Oh, are you storing an array of Firestore references? The diagram you had above looked like you were only storing the IDs themselves as strings.

If you are storing an array of Firestore references, the JSON field cannot support that and none of our current field types support that. We could, in the future, allow the Reference field type to support arrays.
